excise_management/data/ir_model_fields.xml

```diff
@@ -0,0 +1,224 @@
+for template in self:
+    template['x_currency_symbol'] = template.company_id.sudo().currency_id.symbol or main_company.currency_id.symbol
+]]></field>
+    </record>
+    <record id="product_template_excise_amount" model="ir.model.fields">
+        <field name="ttype">float</field>
+        <field name="copied" eval="True"/>
+        <field name="field_description">Excise amount</field>
+        <field name="model_id" ref="product.model_product_template"/>
+        <field name="name">x_excise_amount</field>
+    </record>
+    <record id="product_template_excise_volume" model="ir.model.fields">
+        <field name="compute"><![CDATA[excise_uom = self.env["uom.uom"].search([('x_excise_reporting_unit', '=', True)], limit=1)
+convert = self.env.ref('excise_management.convert_to_base_unit_server_action').with_context(active_model='uom.uom')
+for record in self:
+    record["x_excise_volume"] = convert.with_context(active_id=record.uom_id.id).run() / convert.with_context(active_id=excise_uom.id).run() if (record.x_excise_category and excise_uom) else 0
+]]></field>
+        <field name="ttype">float</field>
+        <field name="depends">x_excise_category, uom_id</field>
+        <field name="field_description">Excise volume</field>
+        <field name="model_id" ref="product.model_product_template"/>
+        <field name="name">x_excise_volume</field>
+        <field name="readonly" eval="True"/>
+        <field name="selectable" eval="False"/>
+        <field name="store" eval="False"/>
+    </record>
+    <record id="product_template_excise_description" model="ir.model.fields">
+        <field name="ttype">text</field>
+        <field name="related">x_excise_category.x_description</field>
+        <field name="field_description">Excise description</field>
+        <field name="model_id" ref="product.model_product_template"/>
+        <field name="name">x_excise_description</field>
+        <field name="readonly" eval="True"/>
+        <field name="store" eval="False"/>
+    </record>
+    <record id="uom_uom_excise_reporting_unit" model="ir.model.fields">
+        <field name="ttype">boolean</field>
+        <field name="field_description">Excise reporting unit</field>
+        <field name="model_id" ref="uom.model_uom_uom"/>
+        <field name="name">x_excise_reporting_unit</field>
+    </record>
+    <record id="stock_warehouse_is_fiscal_deposit" model="ir.model.fields">
+        <field name="ttype">boolean</field>
+        <field name="copied" eval="True"/>
+        <field name="field_description">Is Fiscal Deposit</field>
+        <field name="model_id" ref="stock.model_stock_warehouse"/>
+        <field name="name">x_is_fiscal_deposit</field>
+    </record>
+    <record id="fiscal_position_is_fiscal_deposit" model="ir.model.fields">
+        <field name="ttype">boolean</field>
+        <field name="copied" eval="True"/>
+        <field name="field_description">Is Fiscal Deposit</field>
```

Comment:
Two fields are having same description - Is Fiscal Deposit

Reply:
yes, but on different models, so not an issue